test/registry.js

```diff
@@ -0,0 +1,48 @@
+import { ethers } from "ethers";
+import { TokenRegistry } from "../src/index";
+
+const ERC721 = artifacts.require("ERC721MintableFull");
+
+describe("TokenRegistry", () => {
+  let ERC721Instance;
+  let ERC721Address;
+  let provider;
+  let signer1;
+  let signer2;
+  let signer3;
+
+  const documentStub = {
+    signature: { merkleRoot: "624d0d7ae6f44d41d368d8280856dbaac6aa29fb3b35f45b80a7c1c90032eeb3" }
+  };
+
+  beforeEach("", async () => {
+    ERC721Instance = await ERC721.new("foo", "bar");
+    ERC721Address = ERC721Instance.address;
+    provider = new ethers.providers.Web3Provider(web3.currentProvider);
+    signer1 = provider.getSigner(0);
+    signer2 = provider.getSigner(1);
+    signer3 = provider.getSigner(2);
+  });
+
+  it("should be able to mint", async () => {
+    const tokenRegistryInstance = new TokenRegistry({ contractAddress: ERC721Address, web3Provider: signer1 });
+    const newOwner = await signer2.getAddress();
+    await tokenRegistryInstance.mint(documentStub, newOwner);
+    const currentOwner = await tokenRegistryInstance.ownerOf(documentStub);
+    expect(currentOwner).to.eql(newOwner);
```

Comment:
to.deep.equal ? =)

Reply:
AAAAAAAAAAAAAAAAAAAAHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH